mst/replicas-cli.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Create a read-only replica using Aiven Client 
+excerpt: Create a read-only replica of the service using Aiven Command Line tool for fully managed services on AWS, Azure, or GCP.
+product: mst
+---
+
+# Read-only Replica with Aiven Client
+
+Read-only replicas provide a great way to reduce the load on the primary server
+by enabling read-only queries to be performed against the replica. It is also a
```

Comment:
Can we reword this a little? I had to read it through a couple of times before I got it. I think the difference between "read only replicas" and "read only queries" is what tripped me up.

Reply:
Sure!!!